README.md

```diff
@@ -16,97 +16,102 @@ When the reader has completed this Code Pattern, they will understand how to:
 ## Flow
 
 1. User enters his details and reserves his seat.
-2. Application updates Crunchy Data PostgreSQL tables accordingly.
+2. Application updates the Crunchy Data PostgreSQL tables accordingly.
 3. Application fetches the updated data from the tables.
 4. Application displays the updated seats availability data and booking details data that was feteched from the respective tables. 
 
 ## Pre-requisites
 
 1. [Red Hat Marketplace Account](https://marketplace.redhat.com/en-us/registration/om).
 2. [IBM Managed Red Hat OpenShift Cluster](https://cloud.ibm.com/kubernetes/catalog/create?platformType=openshift).
-3. For all operators being installed from RHM, IBM managed ROKS OpenShift cluster version 4.3 or higher is mandatory. Please set up Classic cluster using the instructions from this URL.
-[Setting up OpenShift Cluster](https://cloud.ibm.com/docs/openshift?topic=openshift-getting-started)
+3. For all operators being installed from Red Hat Marketplace, a Red Hat OpenShift cluster version 4.3 or higher is mandatory. Please set up a classic cluster using the instructions from this URL. [Setting up OpenShift Cluster](https://cloud.ibm.com/docs/openshift?topic=openshift-getting-started)
 4. [OC & kubectl CLI](https://docs.openshift.com/container-platform/3.6/cli_reference/get_started_cli.html).
 
-# Steps
-
-Please follow the below to setup and run this code pattern.
+## Steps
 
 1. [Clone the repo](#1-clone-the-repo)
 2. [Deploy a Crunchy PostgreSQL for Kubernetes Operator to an OpenShift cluster](#2-deploy-a-crunchy-postgresql-for-kubernetes-operator-to-an-openshift-cluster)
 3. [Run the Application](#3-run-the-application)
 4. [Analyse the results](#4-analyse-the-results)
 
-### 1. Clone the repo
+## 1. Clone the repo
+<!--EM: What is included in this repo? Is this the application? If so, I wonder if we should change the title since the reader is not truly building an app, but, rather, connecting an app to a database to track movie ticket reservations-->
```

Comment:
Yes this repo contains a generic applications developed by us. This application will be modified by users to cater to their needs. Hence the title name has build. This is how we used to publicize earlier.  This code pattern also shows them how to connect to CrunchyDB and gives them an idea of how it can be used for various range of applications.